bin/happo-ci

```diff
@@ -34,6 +32,15 @@ if [ -f "yarn.lock" ]; then
   CHECK_FOR_HAPPO="grep 'happo\.io@' yarn.lock"
 fi
 
+resolve-npm-bin() {
+  NPM_BIN=$(${NPM_CLIENT} bin)
+  if [ ! -f "${NPM_BIN}/happo" ]; then
+    # When invoked via `npx run` happo might not be installed locally. In this
+    # case we can use the npx directory (where this script is currently located).
+    NPM_BIN="$(cd "$(dirname "${BASH_SOURCE[0]}" )" > /dev/null 2>&1 && pwd)"
```

Comment:
Could you use the npm-which package here, and set the base dir to process.cwd()?

Reply:
I'll have a look!

Reply:
It doesn't look like npm-which will solve this particular use-case (edge-case really). I'm trying to address an issue coming from using `npx run happo`, where if you don't have `happo.io` listed in package.json you'd run into this issue (from CircleCI): 

```bash
/home/circleci/.npm/_npx/53/bin/happo-ci: line 71: /home/circleci/project/node_modules/.bin/happo: No such file or directory
```

Notice where the `happo-ci` script is located - I don't think npm-which will help in this case.